### PR TITLE
[FIX] base_geoengine_demo readme

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: base_geoengine_demo/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* Damien Crier <damien.crier@camptocamp.com>
+* Cyril Gaudin <cyril.gaudin@camptocamp.com>
+* Pierre Verkest <pverkest@anybox.fr>
+* Benjamin Willig <benjamin.willig@acsone.eu>
+* Devendra Kavthekar <dkatodoo@gmail.com>
+* Emanuel Cino <ecino@compassion.ch>
+* Thomas Nowicki <thomas.nowicki@camptocamp.com>
+* Alexandre Saunier <alexandre.saunier@camptocamp.com>
+* Sandip Mangukiya <smangukiya@opensourceintegrators.com>
+* Daniel Reis <dreis@opensourceintegrators.com>
```
Comment (yvaucher) [inline base_geoengine_demo/readme/CONTRIBUTORS.rst:5]: This list is probably not accurate as not every one that contributed to base_geoengine_demo contributed to base_geoengine.

About your 2 different addresses, I guess that you want to keep your old one whereas noone can contact you that way?

**Review by yvaucher — CHANGES_REQUESTED** [review]

File: base_geoengine_demo/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (dreispt) [reply base_geoengine_demo/readme/CONTRIBUTORS.rst:5]: Fair enough. Did my best approach at that.

**Comment by dreispt** [comment]
Fixed: kept only the names mentioned in the commit history.

**Comment by dreispt** [comment]
Ping @yvaucher

**Comment by dreispt** [comment]
Ping @yvaucher 
Are you OK with the changes?

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
Thanks

**Pull request merged**